src/runtime/traceback.h

```diff
@@ -27,19 +27,25 @@ class GCVisitor;
 extern "C" BoxedClass* traceback_cls;
 class BoxedTraceback : public Box {
 public:
-    std::vector<const LineInfo*> lines;
+    typedef llvm::SmallVector<LineInfo, 1> LinesVector;
+    Box* tb_next;
+    LinesVector lines;
```

Comment:
at this point only when getTraceback() is called.  an option is to just
loop there creating multiple traceback objects.

On Wed, Jun 17, 2015 at 6:34 PM, Kevin Modzelewski <notifications@github.com
> wrote:

> In src/runtime/traceback.h
> <https://github.com/dropbox/pyston/pull/564#discussion_r32692920>:
>
> > @@ -27,19 +27,25 @@ class GCVisitor;
> >  extern "C" BoxedClass* traceback_cls;
> >  class BoxedTraceback : public Box {
> >  public:
> > -    std::vector<const LineInfo*> lines;
> > +    typedef llvm::SmallVector<LineInfo, 1> LinesVector;
> > +    Box* tb_next;
> > +    LinesVector lines;
>
> When will there be more than one line for a traceback object?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dropbox/pyston/pull/564/files#r32692920>.
>